resources/functionMap.php

```diff
@@ -6320,10 +6320,10 @@
 'mb_decode_mimeheader' => ['string', 'string'=>'string'],
 'mb_decode_numericentity' => ['string', 'string'=>'string', 'convmap'=>'array', 'encoding'=>'string'],
 'mb_detect_encoding' => ['string|false', 'str'=>'string', 'encoding_list='=>'mixed', 'strict='=>'bool'],
-'mb_detect_order' => ['bool|list<string>', 'encoding_list='=>'mixed'],
+'mb_detect_order' => ['bool|list<non-falsy-string>', 'encoding_list='=>'mixed'],
```

Comment:
Do this kind of changes actually bring any value at all? I feel like it's counterproductive and might lead to possible mistakes if something is overlooked (i.e. are we 100% sure there can be no falsy string in that list, ever under all circumstances and system setups?).